### DISCUSSION TITLE
Exibir Itens de mão

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#1066] [Opinioes]
### Servidor afetado

Henesys

### Descrição

Quando um totem é equipado, ele sobrescreve o visual do item de mão equipado. Seria legal se funcionasse como os capacetes, onde dá pra equipar e manter o visual cosmético ativo ao mesmo tempo.

- O totem pode continuar na hotbar, para não gerar confusão 
![image](https://github.com/user-attachments/assets/c7c2161c-b305-4878-81e9-49711f530255)

- Como os cosméticos são desativados em mundos de batalha, também não haveria grandes problemas

**Comment by leogianfagna** [comment]
oi @Yokiro27, infelizmente é a forma que o minecraft trata isso. já vimos a respeito mas não tem muito o que fazer
